crates/wasmparser/src/parser.rs

```diff
@@ -151,6 +151,9 @@ pub enum Payload<'a> {
         data_offset: usize,
         /// The actual contents of the custom section.
         data: &'a [u8],
+        /// The range of bytes that specify this whole custom section, specified
+        /// in offsets relative to the start of the byte stream.
```

Comment:
Could this clarify that the intent of this range vs `data_offset` is that this range encompasses the name length and the name in addition to the data?